share/spack/gitlab/cloud_pipelines/stacks/hep/spack.yaml

```diff
@@ -130,12 +130,12 @@ spack:
 
   # CUDA
   #- acts +cuda +traccc cuda_arch=80
-  #- celeritas +cuda ~openmp +vecgeom cuda_arch=80
+  #- celeritas +cuda ~openmp +vecgeom cuda_arch=80 +covfie
```

Comment:
We might also be able to enable this for `~vecgeom` if CUDA RDC was the blocker.